Libraries/Inspector/NetworkOverlay.js

```diff
@@ -10,7 +10,7 @@
 
 'use strict';
 
-import type {RenderItemProps} from '../Lists/VirtualizedList';
+import type {RenderItemProps} from '@react-native/virtualized-lists';
```

Comment:
isn't it also exported from `Libraries/Lists/VitualizedLists` ?
If no, then won't it become a dep for `Inspector/` (at least for now)?

Same for other places inside `Libraries/` ?

Reply:
from my understanding the idea is to extract `VirtualizedList` from react-native, other places are just exporting it for compatibility reasons, so I would say no

Reply:
Yep. That makes sense! Was confused w/ earlier reviews from Nick..
👍